toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/WorldTrackingCameraController.kt

```diff
@@ -109,17 +127,18 @@ internal class WorldTrackingCameraController(private val onLocationDataSourceFai
      *
      * @since 200.7.0
      */
-    private fun updateCamera(location: Location) =
+    private fun updateCamera(location: Location) {
         cameraController.setOriginCamera(
             Camera(
                 location.position.y,
                 location.position.x,
-                if (location.position.hasZ) location.position.z ?: totalElevationOffset else totalElevationOffset,
-                totalHeadingOffset,
+                if (location.position.hasZ) location.position.z!! else 0.0,
```

Comment:
Why is the elevation and heading offset not applied anymore?

Also, better to avoid null asserts:
```suggestion
                if (location.position.hasZ) location.position.z ?: 0.0 else 0.0,
```